packages/x-charts-pro/src/BarChartPro/BarChartPro.tsx

```diff
@@ -22,7 +23,13 @@ export interface BarChartProProps
     Omit<
       ChartContainerProProps<'bar', BarChartProPluginsSignatures>,
       'series' | 'plugins' | 'seriesConfig'
-    > {}
+    > {
+  /**
+   * If true, shows the default chart toolbar.
+   * @default false
+   */
+  showToolbar?: boolean;
```

Comment:
It might be simpler to have it as `toolbar` only. This way you can eventually type it as `toolbar: boolean | ToolbarOptions`

Reply:
Good point. That would break away from the Data Grid's API, though. 

The Data Grid toolbar suggests using composition when we need to customize the toolbar ([toolbar docs](https://mui.com/x/react-data-grid/components/toolbar/#recipes), [export docs](https://mui.com/x/react-data-grid/components/export/#toolbar-export-menu)), so maybe the idea is that options are passed directly to the specific components? E.g., if you want to customize the print dialog, you'd pass some props to the `PrintDialogToolbarButton`. 

Do you think we should break away from the Data Grid pattern this case?

Reply:
Think I'm ok with that